goose3/configuration.py

```diff
@@ -103,7 +102,9 @@ def __init__(self, *, attr=None, value=None, content=None, subcontent=None,
     def __repr__(self):
         if self.tag:
             return f"PublishDatePattern(tag={self.tag}, attr={self.attr}, value={self.value} domain={self.domain})"
-        return f"PublishDatePattern(attr={self.attr}, value={self.value} content={self.content} subcontent={self.subcontent} domain={self.domain})"
+        res = f"PublishDatePattern(attr={self.attr}, value={self.value} content={self.content} " \
```

Comment:
I would even say let's go for a 150 column width. There are def cases where breaking up a long line is great, this doesn't feel like it. additionally, is there a way to use parentheses () instead of forward slash \ ?

Reply:
This would be resolve when running black formatting. I agree that the code shouldn't be updated by the CI, but we could have it "fail" if it doesn't pass black. Once this is done and committed, the black formatting would be my next step.